projects/public-search/src/app/document-detail/request/pickup-location/pickup-location.component.ts

```diff
@@ -111,7 +143,7 @@ export class PickupLocationComponent implements OnInit {
       () => {
         this.requestMessage = {
           success: true,
-          message: this._translateService.instant('A request has been placed on this item.')
+          message: this._translateService.instant('A request has been placed on this ' +  this.recordComponent)
```

Comment:
You can't use this concatenation here because it will cause trouble for string extraction for translate process. You need to use this kind of structure : `this._translateService.instant('A request has been placed on this {type}', { type: this.recordComponent });`

Reply:
I tried, but it does not work. I also tried:
this._translateService.instant('A request has been placed on this {{type}}', { type: this.recordComponent });
This instead seams to work:
this._translateService.instant(`A request has been placed on this ${ this.recordComponent }`)

Reply:
@vgranata It works, but since the translation is missing, you see the untranslated thing. For your test, you can add a string in the translation file.